test/composition.jl

```diff
@@ -132,4 +132,10 @@ using Test, LinearMaps, LinearAlgebra, SparseArrays
     X = rand(size(Ls, 2), 10)
     Y = similar(X, (size(Ls, 1), size(X, 2)))
     @test mul!(Y, Ls, X) ≈ L4.lmap * L3.lmap * L2.lmap * L1.lmap * X
+
+    # test isposdef on a case where sufficient conditions work
+    B = LinearMap([1 0; 0 1]) # isposdef!
```

Comment:
The tests in lines 139-140 fail, because `B` itself is not recognized as positive definite. Those two should be fixed with
```suggestion
    B = LinearMap([1 0; 0 1], isposdef=true) # isposdef!
```